### PR TITLE
kernelci.org: legacy: chromeos: add new supported boards

### PR DESCRIPTION
**Pull request by a-wai**
Depend on https://github.com/kernelci/kernelci-core/pull/2199 and https://github.com/kernelci/kernelci-core/pull/2218

### CODE REVIEW

**Comment by gctucker** [comment]
@a-wai Hello, the other PRs have been merged so is this one ready now?

**Comment by a-wai** [comment]
@gctucker Indeed, I forgot about this one, my bad

**Review by gctucker — APPROVED** [review]
Verified on staging:
https://static.staging.kernelci.org/docs/legacy/instances/chromeos/chromeos_changelog/#supported-boards

File: kernelci.org/content/en/docs/legacy/instances/chromeos/chromeos_changelog.md [hunk 1]
```diff
@@ -33,10 +33,12 @@ Specific instructions on how to fetch and build ChromiumOS from a manifest file
...
 | Board       | Kernels shipped in image | Kernels tested by KernelCI (replacing image kernels during boot) |
 |-------------|:------------:|:-------:|
+| [arcada](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-sarien/20231106.0/amd64/) | default | stable:linux-6.1.y <br> stable:linux-6.6.y |
 | [asurada](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-asurada/20231106.0/arm64) | v6.6.x<br> + panfrost | stable:linux-6.1.y <br> stable:linux-6.6.y |
 | [brya](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-brya/20231106.0/amd64) | default | stable:linux-6.1.y <br> stable:linux-6.6.y |
 | [cherry](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-cherry/20231106.0/arm64) | v6.6.x<br> + 3 display patches<br> + panfrost | stable:linux-6.6.y |
 | [coral](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-coral/20231106.0/amd64) | default | stable:linux-6.1.y <br> stable:linux-6.6.y |
+| [corsola](https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-corsola/20231206.0/arm64) | linux-next 20231106<br> + mtk HW enablement patches<br> + panfrost | |
```
Comment (gctucker) [inline kernelci.org/content/en/docs/legacy/instances/chromeos/chromeos_changelog.md:41]: Note: `corsola` doesn't have any stable branch names listed in the third column.  I guess this is expected if support is still only in linux-next and not mainline yet.

**Pull request merged**